docs/vm/create-windows-vm.md

```diff
@@ -58,18 +58,21 @@ The `bootOrder` values need to be set with the installation image first. If you
 
 1. The **first volume** is an `Image Volume` with the following values: 
     1. `Name`: The value `cdrom-disk` is set by default. You can keep it or change it.
-    2. `Image`: Select the Windows image to be installed. See [Upload Images](../upload-image.md) for the full description on how to create new images.
-    3. `Type`: Select `cd-rom`.
+    2. `Type`: Select `cd-rom`.
+    3. `Image`: Select the Windows image to be installed. See [Upload Images](../upload-image.md) for the full description on how to create new images.
     4. `Size`: The value `20` is set by default. You can change it if your image has a bigger size.
     5. `Bus`: The value `SATA` is set by default. It's recommended you don't change it.
 2. The **second volume** is a `Volume` with the following values:
-    1.  `Name`: The value `rootdisk` is set by default. You can keep it or change it.
-    2.  `Size`: The value `32` is set by default. See the disk space requirements for [Windows Server](https://docs.microsoft.com/en-us/windows-server/get-started/hardware-requirements#storage-controller-and-disk-space-requirements) and [Windows 11](https://docs.microsoft.com/en-us/windows/whats-new/windows-11-requirements#hardware-requirements) before changing this value.
-    3.  `Bus`: The value `VirtIO` is set by default. You can keep it or change it to the other available options, `SATA` or `SCSI`.
+    1. `Name`: The value `rootdisk` is set by default. You can keep it or change it.
+    2. `Type`: Select `disk`.
+    3. `Storage Class`: You can use the default or specify a custom one.
```

Comment:
I think it'd be worthwhile reiterating what the default is here.

Reply:
User can change the default value in the **Advanced>StorageClass** page

Reply:
I understand, but what is the default value @futuretea ? We should mention the default value here.

Reply:
Thanks. Updated in the latest commit: https://github.com/harvester/docs/pull/203/commits/4d20b2264791be76eff51d873e876f475675d579